packages/public/sights/research/data/wireframes360/wireframes360.json

```diff
@@ -0,0 +1,254 @@
+{
+  "fesc20-front-left": {
+    "id": "fesc20-front-left",
+    "category": "exterior",
+    "label": "front-left",
+    "overlay": "fesc20-front-left.svg",
+    "vehicle": "fesc20",
+    "mirror_sight": "fesc20-front-right",
+    "tasks": ["damage_detection"]
```

Comment:
Hmm for me the Wireframe360 are not sights, so you should not have the exact same format as the sights for the JSON. Instead, you could have something like this :

```json
{
  "fesc20-front-left": {
    "id": "fesc20-front-left",
    "label": "front-left",
    "overlay": "fesc20-front-left.svg",
    "vehicle": "fesc20",
    "viewpoint": "front-left"
  }
}
```

This means that you need to update the schema for the JSONs so that it validates this data structure. The viewpoint property is either : front-left / front-right / rear-left / rear-right